src/runtime/runtime-test.js

```diff
@@ -1178,9 +1181,14 @@ describes.realWin('ConfiguredRuntime', {}, env => {
         .expects('addLabels')
         .withExactArgs(['redirect'])
         .once();
-      analyticsMock
+      eventManagerMock
         .expects('logEvent')
```

Comment:
Shouldn't this be .expects('logSwgEvent') since that is the function we are calling in runtime? Although logEvent gets called internally, that implementation might change in the future.

Reply:
done